packages/nextjs/app/Home/builders/page.tsx

```diff
@@ -0,0 +1,126 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { useScaffoldEventHistory, useScaffoldReadContract } from "~~/hooks/scaffold-eth";
+
+const styles = `
+    .glass-effect {
+        background: rgba(255, 255, 255, 0.05);
+        border-radius: 16px;
+        box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+        backdrop-filter: blur(7px);
+        -webkit-backdrop-filter: blur(7px);
+        border: 1px solid rgba(255, 255, 255, 0.1);
+    }
```

Comment:
Try using only Tailwind to specify styles. And if you don't want to repeat yourself, create React components to reuse the style.